modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,17 +1,65 @@
 // Module included in the following assemblies:
 //
-// virt/install/virt-installing-virtctl.adoc
+// virt/install/virt-enabling-virtctl.adoc
 
 [id="virt-installing-virtctl-client_{context}"]
 = Installing the virtctl client
 
-Install the `virtctl` client from the `kubevirt-virtctl` package.
+Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+
+.Prerequisites
+
+* You must have downloaded the `virtctl` client.
 
 .Procedure
 
-* Install the `kubevirt-virtctl` package:
+* For Linux:
+
+. Extract the tarball. The following CLI command extracts it into the same directory as the tarball and is applicable for all distributions:
```

Comment:
Because this step applies to all distributions, I think it makes more sense to move it up one level.

Reply:
@sjhala-ccs Per QE, the tarball step only applies to Linux. See resolved comments here: https://github.com/openshift/openshift-docs/pull/36662#discussion_r714552491